src/main/java/edu/nicholaidudakiwwarrick/advancedjava/model/DatabasePerson.java

```diff
@@ -0,0 +1,118 @@
+package edu.nicholaidudakiwwarrick.advancedjava.model;
+
+import javax.persistence.*;
+/**
+ * This class models a database table that represents a person
+ * in the {@Code @Table} annotation
+ *
+ * @author Nicholai Dudakiw-Warrick
+ */
+@Entity
+@Table(name="person")
+public class DatabasePerson {
```

Comment:
rather than call this class DatabasePerson which is a bit awkward (I think) consider using a package to separate the classes. 
e.g. 
com.origamisoftware.domain.database.Person
Or you could use DTO (which standards for DatabaseTransferObject)